tests/Unit/OutputFormatTest.php

```diff
@@ -1025,4 +1035,133 @@ public function createCompactReturnsInstanceWithRenderCommentsDisabled(): void
 
         self::assertFalse($newInstance->getRenderComments());
     }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsNewOutputFormatInstance(): void
```

Comment:
I think "New" is redundant in the test method name, but it's consistent with `createCompactReturnsNewOutputFormatInstance`, which I signed off :shrug: